slack-base/src/main/java/com/hubspot/slack/client/methods/params/chat/AbstractChatMessageParams.java

```diff
@@ -15,6 +15,7 @@
 public abstract class AbstractChatMessageParams implements MessageParams {
   @JsonProperty("channel")
   public abstract String getChannelId();
+  public abstract String getToken();
```

Comment:
do you need this? I would have expected that given that this implements `HasToken` via `MessageParams`, you shouldn't need to explicitly define it here, right? Failing that, seems like it should have an `@Override` tag